cookiecutter.json

```
@@ -40,7 +42,8 @@
         "license": "Select a license",
         "backend": {
             "__prompt__": "Choose a build backend",
-            "hatch": "Hatchling (recommended for pure Python projects)"
+            "hatch": "Hatchling (recommended for pure Python projects)",
+              "vcs": "Let hatch get the version from your version control system (Git/Mercurial/SVN)?"
```

More scalable language
```suggestion
              "vcs": "Obtain the version from your version control system (Git/Mercurial/SVN)?"
```